CIP-1694/README.md

```diff
@@ -1,530 +1,874 @@
-| 1. Motion of no-confidence                           | :x:                      | $P_1$    | $Q_1$    | None          | $R_1$    |
-| 2(a). New Committee/quorum (_normal state_)          | :heavy_check_mark:       | $P_{2a}$ | $Q_{2a}$ | SPO Vote      | $R_{2a}$ |
-| 2(b). New Committee/quorum (_state of no-confidence_)| :x:                      | $P_{2b}$ | $Q_{2b}$ | None          | $R_{2b}$ |
-| 3. Update to the Constitution                        | :heavy_check_mark:       | $P_3$    | $Q_3$    | SPO Vote      | $R_3$    |
-| 4. Hard-Fork initiation                              | :heavy_check_mark:       | $P_4$    | 0        | -             | $R_4$    |
-| 5. Protocol parameter changes                        | :heavy_check_mark:       | $P_5$    | $Q_5$    | SPO Vote      | $R_5$    |
-| 6(a). Treasury withdrawal, $[T_0, T_1)$              | :heavy_check_mark:       | $P_{6a}$ | $Q_{6a}$ | SPO Vote      | $R_{6a}$ |
-| 6(b). Treasury withdrawal, $[T_1, T_2)$              | :heavy_check_mark:       | $P_{6b}$ | $Q_{6b}$ | SPO Vote      | $R_{6b}$ |
-| 6(c). Treasury withdrawal, $[T_2, T_3)$              | :heavy_check_mark:       | $P_{6c}$ | $Q_{6c}$ | SPO Vote      | $R_{6c}$ |
-
-Some of the parameters given in this table ( $P_1$ ... $R_{6c}$, $T_1$ ... $T_3$ ) may be updatable protocol parameters, but others should be hard-coded. This proposal deliberately leaves both this choice and the choice of actual parameter values open for discussion.
 
 > **Note**
-> For all treasury withdrawals, the withdrawal threshold is the **total** amount of Lovelace that is withdrawn by the action, not the amount of any single withdrawal if the action specifies more than one withdrawal.
+> Any Ada holder may register their stake credential as a DRep and delegate to themselves if they wish to actively participate in
+> voting.
 
-##### Governance Action Lifecycle
+#### Registered DReps
 
-Governance actions are checked for ratification only on an epoch boundary. This delay allows everyone to vote on each proposal and prove that they are active.
+In Voltaire, existing stake credentials will also be
+able to delegate their stake to registered DReps.
+DRep delegation will mimic the existing stake delegation mechanisms (via on-chain certificates).
+Similarly, DRep registration will mimic the existing stake registration mechanisms.
 
-**At most one** governance action **of each type** (i.e. one from each of the six categories listed above) can be staged for enactment in any given epoch. For example, there can only be one treasury withdrawal action in a single epoch (it may, however, comprise many individual withdrawals).
+Registered DReps are identified by a credential that can be either:
 
-Once ratified, actions will be staged for enactment. Actions that have been staged will be enacted on the following epoch boundary, unless they are dropped. All submitted governance actions will therefore either:
+* A verification key (Ed25519)
```

Comment:
Is this any verification key? or is there a defined derivation path  -- I am thinking about the challenge of key discovery for wallets.

Reply:
at the protocol level, there are only verification keys (no derivations). but I agree completely that we should have a standard derivation scheme for this (in a separate CIP).

Reply:
Wallets and key derivations don't exist from the ledger's perspective, so any key.